rikai/public/js/custom.js

```diff
@@ -0,0 +1,568 @@
+//preloading for page
+$(window).on('load', function() { // makes sure the whole site is loaded 
+	var status = $('#status');
+	var preloader = $('#preloader');
+	var body = $('body');
+	status.fadeOut(); // will first fade out the loading animation 
+	preloader.delay(0).fadeOut('fast'); // will fade out the white DIV that covers the website. 
+	body.delay(0).css({'overflow':'visible'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/rikai-academy/book_system_01/pullRequest?prid=7834536)
```suggestion
	body.delay(0).css({"overflow":'visible'});
```